src/Presentation/Loader/PaymentFormAssetLoader.php

```diff
@@ -51,7 +50,7 @@ public function register($controller)
         }
 
         Media::addJsDef([
-            'saferpay_official_ajax_url' => $this->context->link->getModuleLink('saferpayofficial', ControllerName::AJAX),
+            'saferpay_official_ajax_url' => $this->context->getLink()->getModuleLink($this->module->name, ControllerName::AJAX),
```

Comment:
$this->context->link->getModuleLink not working? :thinking:

Reply:
no, because I'm using a different class adapter LegacyContext